### DISCUSSION TITLE
Obsidian Plugin

### DISCUSSION THREAD

**Discussion by Warden20** [Flow-Launcher/Flow.Launcher#802] [Ideas]
This notes program is gaining more relevance with time, and there are already integrations in Mac and Linux operating systems, but I have not seen anything similar compatible with windows. It would be awesome to take advantage of that and make a plugin that has the basic compatibility features.
https://forum.obsidian.md/t/alfred-workflow-for-obsidian/23297
https://forum.obsidian.md/t/linux-alfred-alternative-for-quick-caputure/20203

**Comment by Garulf** [comment]
It seems you're not the only one who wants this feature! I'll start creating one very soon

**Reply by maholli** [reply]
excited for this! thank you

**Comment by broadsword-MO** [comment]
Wow, this could be sweet! I use Obsidian pretty much every day. And that OCR feature looks quite handy.

**Reply by Garulf** [reply]
Heh, I'm not sure where that OCR is from I can't find any mention of it from Obsidian. I'm thinking its MacOS or and Alfred api?

**Comment by Garulf** [comment]
Plugin is released: https://github.com/Garulf/Obsidian-Notes

This is only version 1.0.0 with basic note searching and opening.

**Reply by Warden20** [reply]
It woulde be awesome this new features:
- Search inside obsidian
- Open daily note
- Quick capture to a note
- Search starred notes
- Quickly switch Workspaces
- Compatibility with Quickadd plugin
- Compatibility with Advance URI Scheme

**Reply by Garulf** [reply]
Thanks for the suggestions! I'll take a look as soon as my schedule allows.

**Comment by Warden20** [comment]
This is awesome! It works perfect

**Comment by MysticMaestro** [comment]
I just started using flow launcher and I found the obsidian plugin but it says error installing the plugin. Any solution?

**Reply by taooceros** [reply]
Probably it is just network issue? The installation process is simply download the zip and extract it to the plugin folder.

**Reply by taooceros** [reply]
If you cannot resolve that, you can manually download the release of obsidian plugin and extract it to the plugin folder. (generally will be %appdata%/flowlauncher/Plugins)
